src/kibana/plugins/doc/controllers/doc.js

```diff
@@ -13,17 +13,25 @@ define(function (require) {
     'kibana/index_patterns'
   ]);
 
+  var html = require('text!plugins/doc/index.html');
+
+  var resolveIndexPattern = {
+    indexPattern: function (courier, savedSearches, $route) {
+      return courier.indexPatterns.get($route.current.params.indexPattern);
+    }
+  };
+
   require('routes')
   .when('/doc/:indexPattern/:index/:type/:id', {
-    template: require('text!plugins/doc/index.html'),
-    resolve: {
-      indexPattern: function (courier, savedSearches, $route) {
-        return courier.indexPatterns.get($route.current.params.indexPattern);
-      }
-    }
+    template: html,
+    resolve: resolveIndexPattern
+  })
+  .when('/doc/:indexPattern/:index/:type', {
+    template: html,
+    resolve: resolveIndexPattern
   });
 
-  app.controller('doc', function ($scope, $route, es, timefilter) {
+  app.controller('doc', function ($scope, $route, es, timefilter, $routeParams) {
```

Comment:
Why was `$routeParams` added as a parameter for this function? It doesn't appear to be used...